tests/acceptance/test_performance_issues.py

```diff
@@ -0,0 +1,54 @@
+import datetime
+from unittest.mock import patch
+
+import pytz
+
+from fixtures.page_objects.issue_details import IssueDetailsPage
+from sentry import options
+from sentry.testutils import AcceptanceTestCase, SnubaTestCase
+from sentry.utils import json
+from sentry.utils.samples import load_data
+
+
+class PerformanceIssuesTest(AcceptanceTestCase, SnubaTestCase):
+    def setUp(self):
+        super().setUp()
+        self.org = self.create_organization(owner=self.user, name="Rowdy Tiger")
+        self.team = self.create_team(
+            organization=self.org, name="Mariachi Band", members=[self.user]
+        )
+        self.project = self.create_project(organization=self.org, teams=[self.team], name="Bengal")
+        self.login_as(self.user)
+
+        options.set("performance.issues.all.problem-detection", 1.0)
+        options.set("performance.issues.all.problem-creation", 1.0)
+        options.set("performance.issues.n_plus_one_db.problem-creation", 1.0)
+
+        self.page = IssueDetailsPage(self.browser, self.client)
+        self.dismiss_assistant()
+
+    @patch("django.utils.timezone.now")
+    def test_with_one_performance_issue(self, mock_now):
+        event = load_data("transaction")
+
+        data = json.loads(
+            self.load_fixture("events/performance_problems/n-plus-one-in-django-new-view.json")
+        )
+        event.update({"spans": data["spans"]})
```

Comment:
I'm guessing the timestamps on these spans won't slide, it's been a bit since I've tweaked a transaction's spans in e2e though

Reply:
I was hoping detection would work without futzing with the timestamps, and it did! I counted myself lucky and moved on for now. Relay's validation checks the event's timestamp, but not the spans. This is a little bit of a hack, is it worth it to update all the spans timestamps?

Reply:
I mean if it works, and changing timestamps is non-trivial, ship it, since it's way more coverage than we had before. But this is one of those assumptions that's easy to break when someone adds something relating to time or relative offsets later and just copy and pastes this test. Basically all time should be relative to a singular frozen offset for a test so you simply can't a global/env isolation issue related to the system clock.

Reply:
Yeah that's fair. In this case since I'm setting the system time to the transaction start, so everything is aligned, including the offsets. It'd break down if some other code is checking system time and isn't using `django.utils.timezone.now`, or otherwise really hates events in the past for some reason. I'll add a comment to the test, but ship as-is for now!